docs/reference/migration/migrate_5_0/scripting.asciidoc

```diff
@@ -1,6 +1,54 @@
 [[breaking_50_scripting]]
 === Script related changes
 
+==== Switched Default Language to Painless
```

Comment:
Mention (for people that might not have a lot of history with ES) that this was migrated **from** Groovy to Painless